src/panel/poi_panel.js

```diff
@@ -33,6 +34,7 @@ function PoiPanel(sharePanel) {
   this.card = true
   this.headerPartial = headerPartial
   this.minimalHourPanel = new MinimalHourPanel()
+  this.imagesPanel = new ImagesPanel()
```

Comment:
It should not be necessary to define this panel here, the images block could be handled automatically by the `PoiBlocContainer`